### ISSUE TITLE
Error on no multipart boundary

### ISSUE THREAD

**Issue by jackHedaya** [strapi/strapi#10610]
### Describe the bug

When a header is provided as `multipart/form` without an error boundary, the server returns a `500 Internal Server Error`. Instead perhaps it should send `400 Bad Request`

### Steps to reproduce the behavior

1. Create an endpoint that accepts multipart 
2. Attempt to hit the endpoint with invalid headers

### Screenshots

<img width="1194" alt="Screen Shot 2021-07-15 at 9 48 27 AM" src="https://user-images.githubusercontent.com/20172754/125798938-8687ade2-c525-472c-92a2-afffa2d59cf3.png">

<img width="607" alt="Screen Shot 2021-07-15 at 9 48 58 AM" src="https://user-images.githubusercontent.com/20172754/125799022-e818c862-59f6-4853-85f2-dcb0a614ab7c.png">

### Code snippets

N/A

### System

- Node.js version: 14.15.4
- NPM version: 7.6.3
- Strapi version: 3.6.5
- Database: SQLite
- Operating system: MacOS Big Sur (M1)

**Comment by derrickmehaffy** [comment]
Can you please provide the code you used to create the endpoint? Generally bug reports are reserved for code that is maintained by us unless there is a larger issue that contributed to the problem.

With the information you provided we have no means to reproduce this, so I need some more information.

**Comment by Hejtmus** [comment]
Hello, I experience the same issue.
I created collection type containing some text fields and 1 media field, when I was posting data to this collection type as JSON omitting media everything was OK, but then I tried to upload files from user to media field using FormData error happened.
I was doing it using ["documentation way"](https://strapi.io/documentation/developer-docs/latest/development/plugins/upload.html#upload-file-during-entry-creation).
Sorry if I got something bad, I'm new to Strapi. 
I hope this is enough info, if not, I can provide more info.
